client/lib/domains/suggestions/index.js

```diff
@@ -9,5 +14,19 @@ export const getSuggestionsVendor = ( isSignup = false ) => {
 	if ( isSignup ) {
 		return 'variation4_front';
 	}
-	return 'variation2_front';
+	return 'variation7_front';
+};
+
+/**
+ * Get the premium level to be used for suggestions.
+ *
+ * @param {boolean} [isSignup=false] Whether we are in the signup flow.
+ *
+ * @returns {number} Premium level to supply to the domain suggestions query.
+ */
+export const getSuggestionsPremiumLevel = ( isSignup = false ) => {
```

Comment:
Should we call this premium level or tier? I thought tier was the usual term, but maybe I'm mistaken. Not particularly tied to either of the terms, just want to make sure we use it consistently.

Re-reading the code, I'm not sure I understand the purpose of this "premium level" 🤔 What does it translate to? Right now it's a bit boolean-ish, but I'm guessing as we enable more premium TLDs, it will grow?

Reply:
Thanks for taking a look, @klimeryk!

So this is currently tied to the way our back end suggestions vendor handles premium domain suggestions. There are three options for the value, `0`, `1`, and `2`, so I used this term to represent the fact that we technically have more than two values for the flag. So the value actually has very little to do with tier so much as the way in which premium suggestions might be incorporated.